zubhub_frontend/zubhub/src/assets/js/styles/views/page_wrapper/pageWrapperStyles.js

```diff
@@ -259,8 +259,12 @@ const styles = theme => ({
   },
   footerLinkStyle: {
     color: 'white',
-  },
-  center: {
+    cursor: 'pointer',
+    '&:hover': {
+      color: 'rgb(254 211 42)',
```

Comment:
Instead of using a primary color (zubhub yellow), can we use something which is a lighter version of ZubHub's red color?

Reply:
okay I guess I could make that work

Reply:
![image](https://user-images.githubusercontent.com/29153968/225442397-d39fd13c-57dc-4701-9de8-1c52b85dab41.png)

will this be suitable?

Reply:
Works!